src/main.rs

```diff
@@ -180,6 +183,21 @@ fn handle_wkt(input_str: String, simplification: f64, is_area: bool) -> Vec<Grid
         .collect()
 }
 
+fn handle_polyline(
+    input_str: String,
+    precision: Option<u32>,
```

Comment:
I think it might be simplest to make this something like `precision_str: &str` and parse to `u32` (throwing any errors) inside here similar to what we're doing in `handle_csv` where the names of the latitude and longitude columns are passed directly.

`matches.value_of("precision").unwrap()` as the argument should work since we're already requiring it to be provided if the input type is "polyline"